src/main/java/org/klodnicki/service/AccountService.java

```diff
@@ -56,4 +56,23 @@ public void removeAccount(String pesel) throws NotFoundInDatabaseException {
     private boolean accountExist(String pesel) {
         return accountRepository.findAccountByPesel(pesel).isPresent();
     }
+
+    public void modifyAccount(String peselAccountToModify, String newFirstName, String newSecondName, String newLastName,
+                              String newPesel, String newPhoneNumber, String newEmail, String newAddress)
+            throws IllegalArgumentException, NotFoundInDatabaseException {
+
+        if (!accountExist(peselAccountToModify)) {
+            throw new NotFoundInDatabaseException(Account.class);
+        }
+
+        if (newPesel.length() != 11) {
```

Comment:
DRY rule is broken here. You already have code that checks is. If the logic will change you will now have 2 places where code change will be required.